tests/runtests.pl

```diff
@@ -124,6 +124,9 @@ BEGIN
 require "azure.pm";
 require "appveyor.pm";
 
+# Promote all warnings to fatal
+$SIG{__WARN__} = sub { die @_; };
```

Comment:
shouldn't `use warnings FATAL => 'all';` do the same?

Reply:
[This page](https://www.perlmonks.org/?node_id=931025) says it does the same, yes.